tools/system_libs.py

```diff
@@ -1480,7 +1480,7 @@ class libwasmfs(MTLibrary, DebugLibrary, AsanInstrumentedLibrary):
 
   cflags = ['-fno-exceptions', '-std=c++17']
 
-  includes = ['system/lib/wasmfs']
+  includes = ['system/lib/wasmfs', 'system/lib/pthread']
```

Comment:
This won't be needed once we make the header public so should we wait for that change?

Reply:
I don't think it makes too much difference either way.

Reply:
Agreed, as long as we remember to clean this up if it does land.